packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/steps/Anciennete/store/store.ts

```diff
@@ -155,10 +159,21 @@ const applyGenericValidation = (
   }
 };
 
-const cleanAbsenceDate = (
+const cleanAbsence = (
   absencePeriods: Absence[],
-  data: CommonInformationsStoreSlice
+  data: CommonInformationsStoreSlice,
+  idcc: SupportedCcIndemniteLicenciement | undefined = undefined
 ): Absence[] => {
+  if (idcc) {
+    // clean absence
+    const motifs = new SeniorityFactory()
+      .create(idcc ?? SupportedCcIndemniteLicenciement.default)
```

Comment:
ici tu as forcément une idcc
```suggestion
      .create(idcc)
```

Reply:
Bien vu !